formats/sarifutils/sarifutils_test.go

```diff
@@ -615,3 +616,30 @@ func TestGetInvocationWorkingDirectory(t *testing.T) {
 		assert.Equal(t, test.expectedOutput, GetInvocationWorkingDirectory(test.invocation))
 	}
 }
+
+func TestGetResultFingerprint(t *testing.T) {
```

Comment:
where can we add a test of same finding, in same file - and have two different fingerprints?
(here or https://github.com/jfrog/frogbot/pull/748/files)

Reply:
I will add it to frogbot integration tests